trussworks/react-uswds
# fix: ExtendedNav Tab Order

# Summary

This PR re-orders the primary/secondary elements in the ExtendedNav so that it matches the visual order that a user sees. When a user uses Tabs through the component,  they will iterate through elements in the same order that they appear

## Related Issues or PRs

closes #2641 

## How To Test

Open Storybook locally and go to the Extended Header Page
Press Tab and confirm the first element focused is `Simple line one`
Then followed by
`Simple link two`
The search text box
The search button
Nav Label


### Screenshots (optional)

https://github.com/trussworks/react-uswds/assets/8581332/e21f8bf4-be75-4605-bb8a-d9e6db4271f2
